weblate/settings_test.py

```diff
@@ -26,7 +26,7 @@
 
 from weblate.settings_example import *  # noqa
 
-CI_DATABASE = os.environ.get("CI_DATABASE", "")
+CI_DATABASE = os.environ.get("CI_DATABASE", "postgresql")
```

Comment:
The intention was to explicitly specify database for tests (possibly with other params as well), to avoid accidentally testing against local server.

Reply:
Right, I accidentely committed it. Will remove in further commits. For now I need https://github.com/translate/translate/pull/4068 to be reviewed/merged, before making this one ready :)